schema/test_sample.py

```diff
@@ -0,0 +1,54 @@
+from brain import brain_pb2 as b
+
+SAMPLE_TARGET = {
+    "PluginName": "WaterBalloon",
+    "Location": "Patio",
+    "Port": "West",
+    "CompletelyUnrelatedKey": False,
+}
+FILTERED_OUTPUT = {
+    "PluginName": "WaterBalloon",
+    "Location": "Patio",
+    "Port": "West",
+}
+
+
+def verify(value, msg):
+    """
+    C-style validator
+
+    Keyword arguments:
+    value -- dictionary to validate (required)
+    msg -- the protobuf schema to validate against (required)
+
+    Returns:
+        True: If valid input
+        False: If invalid input
+    """
+    from google.protobuf.message import EncodeError
```

Comment:
your import should be at the top at the file instead of within the function

Reply:
changed